src/argilla/client/feedback/integrations/huggingface/card/argilla_model_template.md

```diff
@@ -0,0 +1,245 @@
+---
+# For reference on model card metadata, see the spec: https://github.com/huggingface/hub-docs/blob/main/modelcard.md?plain=1
+# Doc / guide: https://huggingface.co/docs/hub/model-cards
+{{ card_data }}
+---
+
+# Model Card for {{ model_id | default("Model ID", true) }}
+
+<!-- Provide a quick summary of what the model is/does. -->
+
+{{ model_summary | default("", true) }}
+
+## Model training
+
+Training the model using the `ArgillaTrainer`:
+
+```python
+# Load the dataset if it's uploaded on huggingface hub, or create it from scratch:
+dataset = FeedbackDataset.from_huggingface({% if dataset_name %}"{{ dataset_name }}"{% else %}"..."{% endif %})
```

Comment:
maybe we can show the `FeedbackDataset.from_argilla` if it is not present in the hub?

Reply:
Now if its found on huggingface will assume its loaded `from_huggingface` there, otherwise `from_argilla`. It's on https://github.com/argilla-io/argilla/pull/3857/commits/d92e36804abfc9e2f703a687c9c22898268a7f9d